lib/whitehall_importer.rb

```diff
@@ -1,6 +1,15 @@
 # frozen_string_literal: true
 
 module WhitehallImporter
+  def self.import_multiple_documents(whitehall_export)
+    whitehall_export["documents"].each do |document|
+      WhitehallMigration::DocumentImport.create!(
+        whitehall_document_id: document["document_id"],
+        state: "pending",
+      )
```

Comment:
This feels a bit strange to me as it's coupled to the API response that we do in the rake task so feels somewhat like a helper method for that task than a method in it's own right.